newIDE/app/src/UI/RaisedButtonWithSplitMenu.js

```diff
@@ -15,6 +15,7 @@ type Props = {|
   icon?: React.Node,
   onClick: ?() => void,
   buildMenuTemplate: (i18n: I18nType) => Array<MenuItemTemplate>,
+  style?: Object,
```

Comment:
In general it's good to type as precisely as possible the style object, so that you don't encourage future developers to pass arbitrary styles. In this case:

```
{|
    marginTop?: number,
    marginBottom?: number,
    marginLeft?: number,
    marginRight?: number,
    margin?: number,
    flexShrink?: 0,
  |}
```

This also helps to refactor in the future UI components, because just looking at the type you know "ok, these are all the styles that can be passed, nothing more, so it's manageable I don't need to handle everything in theory"

Reply:
(resolving as this will be done in the next PR)